operator/cmd/syncclusterconfig/sync.go

```diff
@@ -125,3 +135,83 @@ func loadBoostrapYAML(path string) (map[string]any, error) {
 
 	return config, nil
 }
+
+// maybeMergeSuperusers merges the values found in users.txt into our superusers
+// list found in our bootstrap.yaml. This only occurs if an entry for superusers
+// actually exists in the bootstrap.yaml.
+func maybeMergeSuperusers(logger logr.Logger, clusterConfig map[string]any, path string) {
+	if path == "" {
+		// we have no path to a users.txt, so don't do anything
+		logger.Info("--users-txt not specified. Skipping superusers merge.")
+		return
+	}
+
+	superusersConfig, ok := clusterConfig[superusersEntry]
+	if !ok {
+		// we have no superusers configuration, so don't do anything
+		logger.Info("Configuration does not contain a 'superusers' entry. Skipping superusers merge.")
+		return
+	}
+
+	superusers, err := loadUsersTxt(path)
+	if err != nil {
+		logger.Info(fmt.Sprintf("Error reading users.txt file %q: %v. Skipping superusers merge.", path, err))
+		return
+	}
+
+	superusersAny, ok := superusersConfig.([]any)
```

Comment:
Could the type assertion be done strait to string array?
```suggestion
	superusersAny, ok := superusersConfig.([]string)
```

Reply:
unfortunately not, the yaml deserializer deserializes into an `[]any` with every element requiring a type assertion to `string`.